bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_batchgenerator.py

```diff
@@ -677,6 +714,7 @@ def CreateTFDatasets(
     validation_split: float = 0.0,
     max_chunks: int = 0,
     shuffle: bool = True,
+    drop_remainder = True,
 ) -> Tuple[tf.data.Dataset, tf.data.Dataset]:
```

Comment:
This is now false

Reply:
I am not sure what is meant by that, thought we decided on having the parameter set to True as it is safer in case user misses the feature